File: software/backend/src/main/java/ru/scms/back/dto/ClientDashboardDto.java
package ru.scms.back.dto;

import java.util.List;

public record ClientDashboardDto(
        UserDto user,
        String currentStage,
        List<OrderDto> orders,
        List<CaseDto> cases,
        List<CertificateDto> certificates,
        List<StackDto> stacks) {}

File: software/client-portal/src/shared/api/dto/clientDashboardDto.ts
import type { CaseDto } from "./caseDto";
import type { CertificateDto } from "./certificateDto";
import type { OrderDto } from "./orderDto";
import type { StackDto } from "./stackDto";
import type { UserDto } from "./userDto";

export interface ClientDashboardDto {
  user: UserDto;
  currentStage: string;
  orders: OrderDto[];
  cases: CaseDto[];
  certificates: CertificateDto[];
  stacks: StackDto[];
}
